tests/foreman/api/test_host.py

```diff
@@ -1040,25 +1052,35 @@ def test_positive_read_enc_information(
 
     :CaseLevel: Integration
     """
-    update_smart_proxy(module_location, default_smart_proxy)
+    lce = (
+        module_puppet_enabled_sat.api.LifecycleEnvironment()
+        .search(query={'search': f'name={ENVIRONMENT} and organization_id={module_puppet_org.id}'})[
+            0
+        ]
+        .read()
+    )
```

Comment:
Looks like black has done some weird formatting here, please fix it !

Reply:
if black is formatting it this way, there isn't much more that can be done. perhaps splitting that formatted string, but that's about it.

Reply:
Yeah this is one of those situations where we accepted Black, so that's what we take here. We can shorten the line by creating more variables that end up used once, but it is readable, just a bit ugly.

Reply:
```suggestion
        .search(
            query={
                'search': f'name={ENVIRONMENT} and organization_id={module_puppet_org.id}'
            })[0].read())
```
Cant we do somthing like this by ourself? Would black still format this ?

Reply:
dunno, you'd have to stick it in a branch and run Black to see. Likely yes, its going to apply its rules. The only good way to effect it here is to make the value in the dict shorter so it doesn't have to line break.